src/ResourceManager/Network/Commands.Network/help/Add-AzureRmVirtualNetworkSubnetConfig.md

```diff
@@ -105,6 +107,19 @@ Accept pipeline input: True (ByPropertyName)
 Accept wildcard characters: False
 ```
 
+### -PrivateAccessService
+Private Access Service```yaml
```

Comment:
@EvgenyAgafonchikov also, would you mind making the description for this parameter a little more descriptive? You can use the above parameters as an example.